pyproject.toml

```diff
@@ -117,6 +117,17 @@ before-all = "gmake preprocess"
 asyncio_mode = "auto"
 
 
+[tool.ruff]
+line-length = 100
+target-version = "py38"
```

Comment:
- I think `target-version` implies from `project.requires-python` by default.

- Why decided to not use default line-length which is 88?

Reply:
1) you are right https://docs.astral.sh/ruff/settings/#target-version. Done.
2) readability - allowing more content on a single line, more complex expressions are not carried over to the following lines.